test/Swashbuckle.AspNetCore.TestSupport/Fixtures/TypeWithValidationAttributesViaMetadataType.cs

```diff
@@ -17,6 +17,7 @@ public class TypeWithValidationAttributesViaMetadataType
         public string StringWithLength { get; set; }
 
         public string[] ArrayWithLength { get; set; }
+        public string StringWithBase64 { get; set; }
```

Comment:
Please keep the spacing consistent:

```suggestion

        public string StringWithBase64 { get; set; }
```